src/main/resources/application.yml

```diff
@@ -40,6 +40,9 @@ spring:
 launchdarkly:
   sdk-key: "${LAUNCH_DARKLY_SDK_KEY:sdk-c732c882-b5df-4e9b-9ec1-679836b93907}"
 
+launchdarkly:
+  sdk-key: "${LAUNCH_DARKLY_SDK_KEY:sdk-c732c882-b5df-4e9b-9ec1-679836b93907}"
```

Comment:
This is just a repeat of lines 40 & 41

Reply:
Something weird happened with a merge and push - will sort it out